docs/source/peer-chaincode-devmode.rst

```diff
@@ -14,6 +14,23 @@ single running instance can be accessed from multiple channels.
 
 All commands are executed from the ``fabric`` folder.
 
+Setup environment
+-----------------
+Build binaries for orderer, peer and configtxgen.
+::
+    make orderer; make peer; make configtxgen;
```

Comment:
Can't this just be done in a single call? 

`make orderer peer configtxgen`

Reply:
🛠️